packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -46,9 +48,9 @@ export default class QuanticSearchBox extends LightningElement {
    * The maximum number of suggestions to display.
    * @api
    * @type {number}
-   * @defaultValue 5
+   * @defaultValue 7
    */
-  @api numberOfSuggestions = 5;
+  @api numberOfSuggestions = 7;
```

Comment:
we increase this to 7 because numberOfSuggestions = # of suggestions + # of recent queries?
If that is the case I would update at least the description to something like `The maximum number of suggestions and recent search queries to display.`

Reply:
yeah sure the description needs to be updated.